### PR TITLE
Pull request for WAZO-2076-user-pause-resume

### PR DESCRIPTION
**Pull request by fblackburn1**
Depends-on: https://github.com/wazo-platform/wazo-agentd-client/pull/10

### CODE REVIEW

**Comment by fblackburn1** [comment]
recheck

File: wazo_agentd/swagger/api.yml [hunk 1]
```diff
@@ -298,6 +298,28 @@ paths:
           description: Agent is not logged
           schema:
             $ref: '#/definitions/Error'
+  /users/me/agents/pause:
+    post:
+      summary: Logoff the agent of the user holding the authentication token
```
Comment (pc-m) [inline wazo_agentd/swagger/api.yml]: Should be `Pause` instead of `Logoff`

File: wazo_agentd/swagger/api.yml [hunk 2]
```diff
@@ -323,6 +345,28 @@ paths:
           description: Agent is not logged
           schema:
             $ref: '#/definitions/Error'
+  /users/me/agents/resume:
+    post:
+      summary: Logoff the agent of the user holding the authentication token
```
Comment (pc-m) [inline wazo_agentd/swagger/api.yml]: `Logoff` again

**Review by pc-m — APPROVED** [review]

**Comment by pc-m** [comment]
regate

**Pull request merged**